src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ResourceQueriesQueryString.feature.disabled

```diff
@@ -0,0 +1,105 @@
+
+  Background:
+  
+
+  @ignore
+  Scenario: Ensure clients can GET information when querying by valid date
+    Given a school exists
+            | schoolId   | nameOfInstitution                             | gradeLevels               | educationOrganizationCategories          |
+            | 255901001 | Grand Bend High School                         | [ "Tenth grade" ]         | [ "School" ]                             |    
+    And there is a valid AcademicWeek using a valid BeginDate
+            | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+            | string         | 2024-04-09 | 2024-04-09 | 0                      |
+    When sending a GET request to "/ed-fi/academicWeeks?beginDate=<beginDate>"  
+            """
+            """
+    Then it should respond with 200
+    And response contains "weekIdentifier: "string" "
+
+  @ignoreScenario 
+  Scenario: Ensure clients can't GET information when querying by invalid date
+    Given a school exists
+            | schoolId   | nameOfInstitution                             | gradeLevels               | educationOrganizationCategories          |
+            | 255901001 | Grand Bend High School                         | [ "Tenth grade" ]         | [ "School" ]                             |
+    And there is a valid AcademicWeek using an invalid BeginDate   
+            | weekIdentifier | beginDate | endDate    | totalInstructionalDays |
+            | string         | -04-09    | 2024-04-09 | 0                      |
+    When sending a GET request to "/ed-fi/academicWeeks?beginDate=<beginDate>"  
+            """
+            """
+    Then it should respond with 400
+    And response body 
+            """
+            {
+                "detail": "Data validation failed. See 'validationErrors' for details.",
+                "type": "urn:ed-fi:api:bad-request:data",
+                "title": "Data Validation Failed",
+                "status": 400,
+                "correlationId": null,
+                "validationErrors": {
+                    "$.beginDate": [
+                    "The value '<beginDate>' is not valid for BeginDate."
+                    ]
+                }
+            }
+            """
+
+  @ignoreScenario 
+  Scenario: Ensure clients can GET information when querying by date out of specified dates
+    Given a school exists
+            | schoolId   | nameOfInstitution                             | gradeLevels               | educationOrganizationCategories          |
+            | 255901001 | Grand Bend High School                         | [ "Tenth grade" ]         | [ "School" ]                             |
+    And there is an unspecified Begin date
+            | weekIdentifier | beginDate  | endDate    | totalInstructionalDays |
+            | string         | 2027-04-09 | 2027-04-09 | 0                      |
+    When sending a GET request to "/ed-fi/academicWeeks?beginDate=<beginDate>"  
```

Comment:
The query date above needs to be a valid value. It can be any date _other than_ `2027-04-09`.

Reply:
Complete